### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zHPhil93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # breach
 A Browser Game inspired by Cyberpunk 2077.
+
+All earnings go to me!!
```
Comment (Ablu) [inline README.md:4]: change to ablu

Comment (Kugelschieber) [reply README.md:4]: Change to Kugel.

**Review by Ablu — CHANGES_REQUESTED** [review]
see inline comments

**Comment by zHPhil93** [comment]
Find ich ganz schön mies!
Viva la Revolution!

**Pull request closed without merging**